components/nimbus/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="uniffi.nimbus" />
```

Comment:
Also I feel like we wanted to change the name of this package, but I don't recall where we landed on that front.

Reply:
I think we should - I opened https://jira.mozilla.com/browse/SDK-95